test/data/rewards-data/migration/publisher_info_schema_current.txt

```diff
@@ -44,8 +44,8 @@ table|activity_info|activity_info|CREATE TABLE activity_info (publisher_id LONGV
 table|balance_report_info|balance_report_info|CREATE TABLE balance_report_info (balance_report_id LONGVARCHAR PRIMARY KEY NOT NULL,grants_ugp DOUBLE DEFAULT 0 NOT NULL,grants_ads DOUBLE DEFAULT 0 NOT NULL,auto_contribute DOUBLE DEFAULT 0 NOT NULL,tip_recurring DOUBLE DEFAULT 0 NOT NULL,tip DOUBLE DEFAULT 0 NOT NULL)
 table|contribution_info|contribution_info|CREATE TABLE contribution_info (contribution_id TEXT NOT NULL,amount DOUBLE NOT NULL,type INTEGER NOT NULL,step INTEGER NOT NULL DEFAULT -1,retry_count INTEGER NOT NULL DEFAULT -1,created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP, processor INTEGER NOT NULL DEFAULT 1,PRIMARY KEY (contribution_id))
 table|contribution_info_publishers|contribution_info_publishers|CREATE TABLE contribution_info_publishers (contribution_id TEXT NOT NULL,publisher_key TEXT NOT NULL,total_amount DOUBLE NOT NULL,contributed_amount DOUBLE,CONSTRAINT contribution_info_publishers_unique     UNIQUE (contribution_id, publisher_key))
-table|contribution_queue|contribution_queue|CREATE TABLE contribution_queue (contribution_queue_id TEXT PRIMARY KEY NOT NULL,type INTEGER NOT NULL,amount DOUBLE NOT NULL,partial INTEGER NOT NULL DEFAULT 0,created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL)
-table|contribution_queue_publishers|contribution_queue_publishers|CREATE TABLE contribution_queue_publishers (contribution_queue_id TEXT NOT NULL,publisher_key TEXT NOT NULL,amount_percent DOUBLE NOT NULL)
+table|contribution_queue|contribution_queue|CREATE TABLE contribution_queue (contribution_queue_id TEXT PRIMARY KEY NOT NULL,type INTEGER NOT NULL,amount DOUBLE NOT NULL,partial INTEGER NOT NULL DEFAULT 0,created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL, completed_at TIMESTAMP NOT NULL DEFAULT 0)
+table|contribution_queue_publishers|contribution_queue_publishers|CREATE TABLE contribution_queue_publishers (contribution_queue_id TEXT NOT NULL,publisher_key TEXT NOT NULL,amount_percent DOUBLE NOT NULL, completed_at TIMESTAMP NOT NULL DEFAULT 0)
```

Comment:
I think that we don't need `completed_at` here as we have link to `contribution_queue` which already has it and they are updated at the same time

Reply:
Makes sense, removed.